parachains/runtimes/starters/shell/Cargo.toml

```diff
@@ -74,6 +74,7 @@ std = [
 	"cumulus-primitives-core/std",
 	"cumulus-primitives-utility/std",
 	"parachain-info/std",
+#	"parachains-common/std",
```

Comment:
@gilescope 
it is not needed here? because dependency `parachains-common = { path = "../../../common", default-features = false }` is there on the line 42